harness/tests/scripts/ScriptInOtherSourceDir.gdj

```diff
@@ -6,17 +6,17 @@ fqName = ScriptInOtherSourceDir
 relativeSourcePath = otherSourceDir/ScriptInOtherSourceDir.kt
 baseType = Node
 supertypes = [
-    godot.Node,
+	godot.Node,
```

Comment:
Lot of .gdj files with indentation suddenly changed to tab instead of space.

Reply:
Just the first line. They were commited wrongly and i just committed then correctly.
Either the godot editor or intellij breaks the formatting of the first line. I need to investigate this at some point but IMO this is very low priority